### DISCUSSION TITLE
On dropdown filed type option values are not translated

### DISCUSSION THREAD

**Discussion by samuel-clara** [solspace/craft-freeform#1452] [Questions & Answers] [answered]
### What happened?

`|t('freeform')` filter is not applied to dropdown option values making it impossible to translate the content in "Submissions" and "Notifications" and also impossible to use "string variables" (for example "OPTION_1") instead of strings on complex forms.

### How can we reproduce this?

1. create a dropdown field element
2. add values of "OPTION_1", "OPTION_2"...
3. add translations of these strings in freeform.php
4. the form renders with options with value="OPTION_1" and value="OPTION_2"

### Environment

- Freeform Pro 5.5.5
- Craft 5.2.9

**Comment by kjmartens** [comment, marked as answer]
Hi @samuel-clara,

Just to clarify, you can only translate option labels, and that could be done with something like this:

``` twig
<label>My Dropdown Field</label>
<select name="{{ myDropdown.handle }}">
{% for option in myDropdown.options %}
    <option value="{{ option.value }}" {{ option.value in field.value ? "selected" }}>{{ option.label|t('freeform') }}</option>
{% endfor %}
</select>
```

However, the actual value selected will always be stored in the database as the value entered in the form builder for it. The Submissions area of the control panel and any email notifications will not take this into consideration.

**Reply by samuel-clara** [reply]
So it is not possible to have a notification email with the selected values translated in the correct user language? Nor to use "string variables" instead of strings in a dropdown element, did I understand it right?

This unfortunately makes your plugin completely unusable in my case

**Reply by kjmartens** [reply]
You could try adding `|t('freeform')` on the Submission data in the email notification. I am not sure if that will work though.
